lib/logging/addon/logging/route.js

```diff
@@ -46,28 +47,29 @@ export default Route.extend({
       });
     } else if (pageScope === 'project') {
       const project = window.l('route:application').modelFor('authenticated.project').get('project');
-      const [clusterId] = project.get('id').split(':');
+      const projectId = project.get('id');
+      const [clusterId] = projectId.split(':');
```

Comment:
Parsing IDs is bad.  The point of getting the project object above was that you could get the clusterId from it directly (`get(project,'clusterId')`, or `get(project,'cluster')` if you want the actual object)